### ISSUE TITLE
How about full, update and revision numbers. 4.6.1.69 etc.

### ISSUE THREAD

**Issue by codeSasquatch** [shanselman/SmallestDotNet#56]
How about full, update and revision numbers. 4.6.1.69 etc.

**Comment by marcosnz** [comment]
@codeSasquatch do you know a way of reliably obtaining this?

**Comment by codeSasquatch** [comment]
The only way I know of is to look inside windows at specific binaries e.g.C:\Windows\Microsoft.NET\Framework64\v4.0.30319\clr.dll 
![image](https://cloud.githubusercontent.com/assets/2086285/10397901/54857724-6e67-11e5-8024-49ecbb3dbac7.png)

**Comment by marcosnz** [comment]
Yeah, me to, but ([in the past, at least](http://blogs.msdn.com/b/rodneyviana/archive/2014/12/23/identifying-the-net-version-you-are-running-2-0-4-5-4-5-1-or-4-5-2.aspx)) this hasn't been terribly reliable:
>Below is a list with the approximate build range for each version:
.NET Version|Clr.dll Build 4.0.30319.<Build>
------------------|------------------------------------------
.NET 4.0|4.0.30319.0 to 4.0.30319.17000
.NET 4.5|4.0.30319.17001 to 4.0.3019.18400
.NET 4.5.1|4.0.30319.18401 to 4.0.30319.34000
.NET 4.5.2|From 4.0.30319.34000

I think it may confuse some people to say "Your version is 4.5.2, your CLR version is 4.0.30319.34000".
What do you think?

**Comment by codeSasquatch** [comment]
I think that Microsoft should provide a registry key with the latest "Product Version" with the revision and build numbers, so that engineers who do installation and also the few who even care about the version of .Net can find it easily. In a seperate key provide the "CLR" version It seems like an oversight to me that could easily be fixed. While its easy enough to go digging through the file structure to find a binary to look at it is not so clear for the beginning engineer or someone who has never been tasked to find the version to find it.

**Comment by binki** [comment]
@codeSasquatch How do you get `4.6.1.69` from `4.6.96.0`? Also, on my system, clr.dll fileversion is `4.6.2006.0`. What version of .net would that mean I have?
